src/main/resources/feedbackQuestionRubricResultStatsTemplate.html

```diff
@@ -1,10 +1,17 @@
 <div class="panel-body">
   <div class="row">
-    <div class="col-sm-4 text-color-gray">
+    <div class="col-sm-8 text-color-gray">
       <strong>
         ${statsTitle}
       </strong>
     </div>
+    <div class="col-sm-4 text-right ${isExcludingSelfOptionAvailable}">
+      <input type="checkbox" id="excluding-self-check" onchange="switchExcludingSelfResultsForRubricStatistics(this)">
```

Comment:
use `onclick` instead. Also, the id should be unique in one HTML file.

Reply:
What can I do to make the `id` unique in an html file?

Reply:
On second thought, the question number thing might not work instructor is viewing results of each student. 
Do you have any suggestion as to what can make the id unique?

Reply:
Can use class name and make use of the .cloest() method in JQuery